src/style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 20px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+  border: 1px solid rgb(230, 230, 230);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background-color: rgb(250, 250, 250);
+}
+
+.input:focus {

```

Comment:
According to Figmas design, `font-weight` should change when the bar is in focus.